### DISCUSSION TITLE
Allow multiple questions in a single Q&A discussion

### DISCUSSION THREAD

**Discussion by robertogl** [community/community#3585] [Discussions]
I think it might be useful being able to ask more questions, and therefore having different threads, within the same Q&A discussion to avoid starting many discussions related to the same argument (issue, file, doc, etc.). 

Thanks!

**Comment by PranamBhat** [comment]
Hi @robertogl 

It is going to be big mess!!!

**Reply by robertogl** [reply]
I do not think it would be any messier than  open ended discussions.
